resource/systemd/unit/preparer.go

```diff
@@ -121,7 +112,22 @@ func (p *Preparer) Prepare(render resource.Renderer) (resource.Task, error) {
 		StartMode:     systemd.StartMode(sm),
 		Timeout:       t,
 	}
-	return unit, unit.Validate()
+	return unit, Validate(unit)
+}
+
+var validUnitFileStates = systemd.UnitFileStates{systemd.UFSEnabled, systemd.UFSEnabledRuntime, systemd.UFSLinked, systemd.UFSLinkedRuntime, systemd.UFSMasked, systemd.UFSMaskedRuntime, systemd.UFSDisabled}
+
+func Validate(t *Unit) error {
```

Comment:
this should go *inline* into Prepare. It'll make more sense why we're doing that after #321